kuzzleio/kuzzle-admin-console
# Remove unauthorized actions

## What does this PR do ?

This PR removes unauthorized actions from the admin console, at least for read-only access.

### How should this be manually tested?

1. Create a read-only user with the following role: 

```
{
  "controllers": {
    "auth": {
      "actions": {
        "getCurrentUser": true,
        "getMyCredentials": true,
        "getMyRights": true,
        "logout": true
      }
    },
    "collection": {
      "actions": {
        "getMapping": true,
        "list": true
      }
    },
    "document": {
      "actions": {
        "count": true,
        "get": true,
        "mGet": true,
        "scroll": true,
        "search": true
      }
    },
    "index": {
      "actions": {
        "list": true
      }
    }
  }
}
```
2. log in to the admin console
3. check ~no write action is available from~ all write actions are greyed out in the interface
